CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+2.1.1
+-----
+
+* Implemented redirect route feature for the Doctrine ORM model 
+* Removed the name for staticPrefix index on the Route model due to name colision when the model is inherited by RediretRoute model
```

Comment:
s/Removed the name for/Removed the explicit name for/
s/due to name/to avoid the name/
s/colision/collision/
s/RediretRoute/RedirectRoute/

Reply:
Thanks :)